### PR TITLE
ACMS-3473: Fix failing cypress taxonomy test.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #[ACMS-3473](https://acquia.atlassian.net/browse/ACMS-3473)

**Proposed changes**
Fix failing cypress tests.

**Alternatives considered**
NA

**Testing steps**
Follow from ticket

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: .github/workflows/acquia_cms_cron.yml [hunk 1]
```diff
@@ -384,8 +384,9 @@ jobs:
 
             # Remove drupal/core patch as it's needed for latest Drupal Core only.
             composer config extra.patches.drupal/core {} --json
-            sed -i 's/2598.patch",/2599.patch"/' modules/acquia_cms_common/composer.json
+            sed -i 's/2598.patch",/2598.patch"/' modules/acquia_cms_common/composer.json
             sed -i '/3356894-mr_3896.patch/d' modules/acquia_cms_common/composer.json
+            sed -i '/3370946-page-title-backport-10-2-2_0.patch/d' modules/acquia_cms_common/composer.json
```
Comment (deepakmishra2) [inline .github/workflows/acquia_cms_cron.yml:389]: i think we can remove this change , once this gets merged or vice versa - https://github.com/acquia/acquia_cms/pull/1730/files

**Review by deepakmishra2 — APPROVED** [review]

File: .github/workflows/acquia_cms_cron.yml (hunk 1 above)
Comment (rajeshreeputra) [reply .github/workflows/acquia_cms_cron.yml:391]: The addition of above change necessary as it is not covered in the mentioned PR.

**Review by apathak18 — APPROVED** [review]

File: .github/workflows/acquia_cms_cron.yml [hunk 2]
```diff
@@ -321,36 +321,10 @@ jobs:
       matrix:
         orca-job:
           - ISOLATED_TEST_ON_CURRENT
-          - INTEGRATED_TEST_ON_LATEST_LTS
+          - INTEGRATED_TEST_ON_PREVIOUS_MINOR
```
Comment (chandan-singh7929) [inline .github/workflows/acquia_cms_cron.yml:324]: Why we have removed `INTEGRATED_TEST_ON_LATEST_LTS` and using `INTEGRATED_TEST_ON_PREVIOUS_MINOR` ?

**Review by chandan-singh7929 — APPROVED** [review]

File: .github/workflows/acquia_cms_cron.yml (hunk 2 above)
Comment (chandan-singh7929) [reply .github/workflows/acquia_cms_cron.yml:324]: As per our discussion, drupal don't have LTS hence we are using previous_minor.

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**